packages/block-editor/src/components/keyboard-shortcuts/index.js

```diff
@@ -0,0 +1,157 @@
+			{
+				modifier: 'access',
+				character: 'z',
+			}
+		);
+
+		registerShortcut(
+			'core/block-editor/insertBefore',
+			'block',
+			__( 'Insert a new block before the selected block(s).' ),
+			{
+				modifier: 'primaryAlt',
+				character: 't',
+			}
+		);
+
+		registerShortcut(
+			'core/block-editor/insertAfter',
+			'block',
+			__( 'Insert a new block after the selected block(s).' ),
+			{
+				modifier: 'primaryAlt',
+				character: 'y',
+			}
+		);
+
+		registerShortcut(
+			'core/block-editor/deleteMultiSelection',
+			'block',
+			__( 'Remove the multi-selected blocks.' ),
+			{
+				character: 'del',
+				aliases: [
+					'backspace',
+				],
+			}
+		);
+
+		registerShortcut(
+			'core/block-editor/selectAll',
+			'selection',
+			__( 'Select all text when typing. Press again to select all blocks.' ),
+			{
+				modifier: 'primary',
+				character: 'a',
+			}
+		);
+
+		registerShortcut(
+			'core/block-editor/unselect',
+			'selections',
+			__( 'Clear selection.' ),
+			{
+				character: 'escape',
+			}
+		);
+	}, [ registerShortcut ] );
+
+	// Shortcuts Logic
+	const { clientIds, rootBlocksClientIds } = useSelect( ( select ) => {
+		const { getSelectedBlockClientIds, getBlockOrder } = select( 'core/block-editor' );
+		return {
+			clientIds: getSelectedBlockClientIds(),
+			rootBlocksClientIds: getBlockOrder(),
+		};
+	}, [] );
+
+	const {
+		duplicateBlocks,
+		removeBlocks,
+		insertAfterBlock,
+		insertBeforeBlock,
+		multiSelect,
+		clearSelectedBlock,
+	} = useDispatch( 'core/block-editor' );
+
+	// Prevents bookmark all Tabs shortcut in Chrome when devtools are closed.
+	// Prevents reposition Chrome devtools pane shortcut when devtools are open.
+	useShortcut( 'core/block-editor/duplicate', useCallback( ( event ) => {
+		event.preventDefault();
+		duplicateBlocks( clientIds );
+	}, [ clientIds, duplicateBlocks ] ), { bindGlobal: true } );
+
+	// Does not clash with any known browser/native shortcuts, but preventDefault
+	// is used to prevent any obscure unknown shortcuts from triggering.
+	useShortcut( 'core/block-editor/remove', useCallback( ( event ) => {
+		event.preventDefault();
+		removeBlocks( clientIds );
+	}, [ clientIds, removeBlocks ] ), { bindGlobal: true } );
+
+	// Does not clash with any known browser/native shortcuts, but preventDefault
+	// is used to prevent any obscure unknown shortcuts from triggering.
+	useShortcut( 'core/block-editor/insertAfter', useCallback( ( event ) => {
+		event.preventDefault();
+		insertAfterBlock( last( clientIds ) );
+	}, [ clientIds, insertAfterBlock ] ), { bindGlobal: true } );
+
+	// Prevent 'view recently closed tabs' in Opera using preventDefault.
+	useShortcut( 'core/block-editor/insertBefore', useCallback( ( event ) => {
+		event.preventDefault();
+		insertBeforeBlock( first( clientIds ) );
+	}, [ clientIds, insertBeforeBlock ] ), { bindGlobal: true } );
+
+	useShortcut( 'core/block-editor/deleteMultiSelection', useCallback( () => {
```

Comment:
I haven't looked two much at why, but `deleteMultiSelection` and `unselect` don't seem to be working. `selectAll` is working, but not because of the code in this file, the callback isn't triggered so it must also be handled elsewhere.

It's noticeable that they're the three shortcuts using a target, so probably related to that, maybe the events aren't being triggered for that element.

Reply:
I don't know, it's probably the same in master but I'll take a look.

Reply:
This is fixed, initially, I misunderstood how the KeyboardShortcuts component work. It only uses a  target when it's used with "children" prop which was not the case here. I removed the target.